packages/inter-protocol/src/collectFees.js

```diff
@@ -14,9 +16,10 @@ export const makeMakeCollectFeesInvitation = (
 ) => {
   const collectFees = seat => {
     const amount = feeSeat.getAmountAllocated(keyword, feeBrand);
-    feeSeat.decrementBy(harden({ [keyword]: amount }));
-    seat.incrementBy(harden({ Fee: amount }));
-    zcf.reallocate(seat, feeSeat);
+    atomicReallocate(
```

Comment:
I think, in the context of other suggestions, `atomicReallocate` should be `atomicTransfer`, since the semantics of each parameter is to decrement from the first of the pair and increment the second.

Reply:
Done